docs/developer/other/IDE_INTEGRATION.md

```diff
@@ -0,0 +1,31 @@
+# IDE Integration
+
```

Comment:
Sounds good, I've added a brief summary of benefits at the top of `IDE_INTEGRATION.md`.